src/guarneri/instrument.py

```diff
@@ -353,16 +386,20 @@ def load(
         config_file = Path(config_file)
         # Load the instrument from config files
         old_classes = self.device_classes
+        old_ignored = self.ignored_classes
+        # Parse device configuration files
+        device_defns = self.parse_config(config_file)
         # Temprary override of device classes
         if device_classes is not None:
             self.device_classes = device_classes
+        if ignored_classes is not None:
+            self.ignored_classes = ignored_classes
```

Comment:
Aren't you already doing this in line 79? 

```
  self.ignored_classes = ignored_classes
```

Reply:
Kind of. Line 79 happens during ``__init__``, so you can set the instrument to *always* ignore these classes. Line 396 is in ``load()``, so you can ignore these classes only for the specific configuration file.